### PR TITLE
Launch file for combined spawn_model + bridge

### PR DESCRIPTION
**Pull request by caguero**
# 🎉 New feature

## Summary

This patch adds two launch files: one for spawning a gz sim model and the other for spawning a gz sim model with a ros_gz bridge.

### How to test it?

Launch gz sim:
```
ros2 launch ros_gz_sim gz_sim.launch.py gz_args:=empty.sdf
```

Hit play.

Use the provided `gz_spawn_model.launch.py` to spawn a vehicle (change `file` accordingly):
```
ros2 launch ros_gz_sim gz_spawn_model.launch.py world:=empty file:=/home/caguero/ros_gz_ws/src/ros_gz/ros_gz_sim_demos/models/vehicle/model.sdf name:=my_vehicle x:=5.0 y:=5.0 z:=0.5
```

Confirm that no container has been created:
```
caguero@cold:~/ros_gz_ws$ ros2 component list

```

Now let's spawn another model with a ros_gz bridge (update `file` and `config_file` accordingly):
```
ros2 launch ros_gz_sim ros_gz_spawn_model.launch.py world:=empty file:=/home/caguero/ros_gz_ws/src/ros_gz/ros_gz_sim_demos/models/vehicle/model.sdf name:=my_vehicle_2 x:=2.0 y:=2.0 z:=0.5 config_file:=/home/caguero/ros_gz_ws//src/ros_gz/ros_gz_bridge/test/config/full.yaml use_composition:=True
```

A second vehicle should be loaded into the scene.

gzserver should be running. Verify that you also see the `/ros_chatter` topic:
```
caguero@cold:~/ros_gz_ws$ ros2 topic list
/parameter_events
/ros_chatter
/rosout
```

And confirm that a container has been created with both nodes:
```
caguero@cold:~/ros_gz_ws$ ros2 component list
/ros_gz_container
  1  /ros_gz_bridge
```

You could also explore launching another vehicle without the `use_composition` parameter.

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [ ] Added example and/or tutorial
- [ ] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [ ] Consider updating Python bindings (if the library has them)
- [ ] `codecheck` passed (See [contributing](https://gazebosim.org/docs/all/contributing#contributing-code))
- [ ] All tests passed (See [test coverage](https://gazebosim.org/docs/all/contributing#test-coverage))
- [ ] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Agazebosim+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: ros_gz_sim/CMakeLists.txt [hunk 1]
```diff
@@ -121,7 +121,10 @@ install(FILES
 
 install(FILES
   "launch/gz_server.launch.py"
+  "launch/gz_spawn_model.launch.py"
   "launch/ros_gz_sim.launch.py"
+  "launch/ros_gz_spawn_model.launch.py"
+
```
Comment (ahcorde) [inline ros_gz_sim/CMakeLists.txt]: remove this empty line

File: ros_gz_sim/launch/gz_spawn_model.launch.py [hunk 2]
```diff
@@ -0,0 +1,99 @@
+
```
Comment (ahcorde) [inline ros_gz_sim/launch/gz_spawn_model.launch.py]: REmove this empty line

File: ros_gz_sim/launch/gz_spawn_model.launch.py [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+    declare_name_cmd = DeclareLaunchArgument(
+        'name', default_value=TextSubstitution(text=''),
+        description='Name of the entity'
+    )
+    declare_allow_renaming_cmd = DeclareLaunchArgument(
+        'allow_renaming', default_value='False',
+        description='Whether the entity allows renaming or not'
+    )
+
+    load_nodes = GroupAction(
```
Comment (ahcorde) [inline ros_gz_sim/launch/gz_spawn_model.launch.py]: This is just one Node, you don't need to use here GroupAction.

File: ros_gz_sim/launch/ros_gz_spawn_model.launch.py [hunk 4]
```diff
@@ -0,0 +1,155 @@
+
```
Comment (ahcorde) [inline ros_gz_sim/launch/ros_gz_spawn_model.launch.py]: Remove empty line

**Review by ahcorde — CHANGES_REQUESTED** [review]

File: ros_gz_sim/CMakeLists.txt (hunk 1 above)
Comment (caguero) [reply ros_gz_sim/CMakeLists.txt]: Removed in [54c68d7](https://github.com/gazebosim/ros_gz/pull/534/commits/54c68d7f3ee5daeba6c9735129fe9c08a0613e42)

File: ros_gz_sim/launch/gz_spawn_model.launch.py (hunk 2 above)
Comment (caguero) [reply ros_gz_sim/launch/gz_spawn_model.launch.py]: Removed in [54c68d7](https://github.com/gazebosim/ros_gz/pull/534/commits/54c68d7f3ee5daeba6c9735129fe9c08a0613e42)

File: ros_gz_sim/launch/gz_spawn_model.launch.py (hunk 3 above)
Comment (caguero) [reply ros_gz_sim/launch/gz_spawn_model.launch.py]: Tweaked in [54c68d7](https://github.com/gazebosim/ros_gz/pull/534/commits/54c68d7f3ee5daeba6c9735129fe9c08a0613e42)

File: ros_gz_sim/launch/ros_gz_spawn_model.launch.py (hunk 4 above)
Comment (caguero) [reply ros_gz_sim/launch/ros_gz_spawn_model.launch.py]: Removed in [54c68d7](https://github.com/gazebosim/ros_gz/pull/534/commits/54c68d7f3ee5daeba6c9735129fe9c08a0613e42)

**Review by ahcorde — APPROVED** [review]

**Pull request merged**